js/classes/entities/midground/enemies/village/bunny.js

```diff
@@ -82,13 +67,13 @@ class Bunny {
         // HOW SHOULD WE MOVE HIM BASED ON HIS CONDITIONS?
         const mult = (this.facing === "left") ? -1 : 1;
         if (this.action === "hopping") {
-            this.x += mult * Bunny.SPEED * GAME.clockTick;
+            this.pos = Vector.add(this.pos, new Vector(mult * Bunny.SPEED * GAME.clockTick, 0));
         }
     };
 
     /** Draw the Bunny on the canvas. */
     draw() {
```

Comment:
Bunny was made for the animation assignment, and does not draw based on the position of the camera yet.